CUDADataFormats/SiPixelCluster/interface/SiPixelClustersCUDA.h

```diff
@@ -5,16 +5,29 @@
 #include "HeterogeneousCore/CUDAUtilities/interface/host_unique_ptr.h"
 #include "HeterogeneousCore/CUDAUtilities/interface/cudaCompat.h"
 
+#include "DataFormats/SoATemplate/interface/SoALayout.h"
```

Comment:
`CUDADataFormats/SiPixelCluster/BuildFile.xml` needs
```xml
<use name="DataFormats/SoATemplate" source_only="1"/>
```
because of this `#include` (same for other `CUDADataFormats` packages in this PR).